term/fhir-term/src/main/java/com/ibm/fhir/term/service/FHIRTermService.java

```diff
@@ -737,7 +737,17 @@ public ValidationOutcome validateCode(CodeSystem codeSystem, CodeableConcept cod
                 return outcome;
             }
         }
-        return validateCode(null, false, null);
+
+        StringBuffer message = new StringBuffer()
```

Comment:
Any Reason for the synchronized class and not StringBuilder?

Reply:
lol, just dumb mistake...will fix

Reply:
fixed